newsfragments/33.feature.rst

```diff
@@ -0,0 +1 @@
+The Trio main loop now uses a higher-precision clock by default.
```

Comment:
This should say "on Windows", since that's the only platform affected.

Reply:
My reasoning is: we use a different *function* on all platforms, but on most platforms this doesn't change behavior at all, so it's not of interest to end users. The only place where *precision* changes is on Windows.